### DISCUSSION TITLE
Cards: Summary of "up to"

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#19] [Ideas]
The phrase "up to" is removed from the following. by "no own" I mean the player that gets to perform the Event, not necessarily the one playing it.

Maastrich Treaty ("as much as possible" principle)
Neocolonialism (this initially had "may" changed to "up to", it's any Influence, but the chance a player has no country where they can't place their own Influence is next to none, so this is fine?)
Rupert Murdoch ("as much as possible" principle)
Shock doctrine (can pick the first choice, leading to a no-op at worst)
Uncomfortable democracies (can pick the first choice, leading to a no-op at worst)
Wolfowitz doctrine ("as much as possible" principle, no own downside)
ALBA ("as much as possible" principle, no own downside)
Austerity plans ("as much as possible" principle)

"Up to" is kept on:

Arab Spring (the first choice is removing, not adding, and the likelihood of having nothing to remove from either of two countries are higher. thus it might be safer to allow "up to" from the other three countries as to not force removing the players own Influence?)

Maybe keep?

Economic rescue (West player gets to add West Influence, possibly forced to add the other West players Influence? on the other hand, all of south america and 7 more countries is plenty?)

**Comment by rodrigoSantamaria** [comment]
The Neocolonialism wording is ok, effectively the chances of not having any influence are close to zero.
It's the opposite on Arab Spring, as you said, so i think is ok to keep the 'up to' there.
The rest is ok.

About Economic Rescue, you gain 2 influence in just 1 of the countries in the list, not on all of them. The list includes all South American countries and another 7, all of which had an economic rescue in the period. I'd leave it as 'Add 2 Influence in a single one of the following countries:...'. It was not the intention on the card for force to add Influence for the other West player.

**Reply by RebelusQuo** [reply]
My current wording for Economic Rescue is:
> Add 2 West Influence in one country among South American countries, South Korea, Indonesia, Thailand, Philippines, Greece, Italy, and Portugal/Spain.

**Comment by rodrigoSantamaria** [comment]
I's say just 'add 2 influence', as it is a text that would execute a west player, no? The 'West' is redundant, and may lead to confusion.

**Reply by RebelusQuo** [reply]
Ok, so we change the effect of this card then. Acknowledged.
